exercises/crypto-square/.dub/dub.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
excuse me for barging in: is this file necessary to have in this repo? what does it do? sorry for my ignorance

Reply:
Oops, it shouldn't be there. I've fixed it in #55